Cargo.toml

```diff
@@ -13,7 +13,7 @@ name = "bindgen"
 readme = "README.md"
 repository = "https://github.com/servo/rust-bindgen"
 documentation = "https://docs.rs/bindgen"
-version = "0.20.1"
+version = "0.20.2"
```

Comment:
`quasi_codegen` is only used in a build script and so not exposed to dependencies in public APIs. Therefore this is not a breaking change.